sda-commons-dependencies/build.gradle

```diff
@@ -82,6 +82,9 @@ dependencies {
     api "net.javacrumbs.json-unit:json-unit-core:$jsonUnitVersion", {
       because "conflict between com.github.tomakehurst:wiremock-jre8 and net.javacrumbs.json-unit:json-unit-assertj"
     }
+    api "org.eclipse.jetty:jetty-http:9.4.52.v20230823", {
+      because "vulnerability CVE-2023-40167 in 9.4.51.v20230217"
+    }
```

Comment:
Don't we want to align the versions of all Jetty modules? Compare how [Dropwizard defines the dependency](https://github.com/dropwizard/dropwizard/blob/release/2.1.x/dropwizard-dependencies/pom.xml):
```
<dependency>
                <groupId>org.eclipse.jetty</groupId>
                <artifactId>jetty-bom</artifactId>
                <version>${jetty.version}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
```